src/App.tsx

```diff
@@ -2,6 +2,7 @@ import { Button } from "@mantine/core"
 import Home from "./Home"
```

Comment:
try setting the project up to accept absolute paths for imports, doesn't matter too much for these pages but will come into play more as the app gets bigger

Reply:
This one I thought would be a two second thing, but actually having issues with it. was basically trying vairations on this: https://dev.to/andrewezeani/how-to-create-absolute-imports-in-vite-react-app-a-step-by-step-guide-28co  seems like it should be working but I am getting module/type declaration errors. What would be the issue here? I mostly tried playing with vite.config, but maybe there is more to do that I don't know about?

Reply:
So there's two factors at play, one is with Vite knowing where to look but there's also a factor with typescript knowing where to look to. I think what will probably solve it is if you added baseUrl property into tsconfig

Reply:
Ah, gotcha, kk, yeah I got this one working. had to add 
"@/*": [
        "*"
      ]
to path. If I understand it, @/ anything is what is mapping to ./src, and the "*" means that anything in there can be mapped that way so long as src is parent. correct?